File: backend/PersonApi/Data/Person.cs
namespace PersonApi.Data;

public class Person
{
    public long Id { get; set; }
    public string FirstName { get; set; } = "";
    public string LastName { get; set; } = "";
    public DateOnly BirthDate { get; set; }
    public string? Address { get; set; }
    public DateTime CreatedAt { get; set; }
}

File: frontend/src/types/person.ts
export interface Person {
  id: number
  firstName: string
  lastName: string
  birthDate: string // ISO yyyy-mm-dd
  age: number
  address: string | null
}

export interface CreatePersonRequest {
  firstName: string
  lastName: string
  birthDate: string // yyyy-mm-dd
  address: string | null
}
